doc/man3/PKCS12_SAFEBAG_get0_attrs.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+PKCS12_SAFEBAG_get0_attrs,PKCS12_get_attr_gen - Retrieve attributes from a PKCS#12 safeBag
```

Comment:
It seems there should be a space after the comma.

Reply:
Fixed this and another occurrence.